openfisca_us/variables/irs/outputs.py

```diff
@@ -403,7 +403,39 @@ class c02500(Variable):
     value_type = float
     entity = TaxUnit
     definition_period = YEAR
+    label = "Taxable social security benefits"
     documentation = """Social security (OASDI) benefits included in AGI"""
+    unit = "currency-GBP"
+
+    def formula(tax_unit, period, parameters):
+        ss = parameters(period).irs.social_security.taxability
+        ymod = tax_unit("ymod", period)
+        mars = tax_unit("mars", period)
+
+        lower_threshold = ss.threshold.lower[mars]
+        upper_threshold = ss.threshold.upper[mars]
+
+        under_first_threshold = ymod < lower_threshold
+        under_second_threshold = ymod < upper_threshold
+
+        e02400 = tax_unit("filer_e02400", period)
+        return select(
+            [
+                under_first_threshold,
+                under_second_threshold,
+                True,
+            ],
+            [
+                0,
+                ss.rate.lower * min_(ymod - lower_threshold, e02400),
```

Comment:
Could you name this a new Python variable `amount_if_under_second_threshold` (and save for the over one)?